utils/glam_subdags/generate_query.py

```diff
@@ -136,13 +139,17 @@ def generate_and_run_glean_task(
     if env_vars is None:
         env_vars = {}
 
+    query_name = task_name.split("_sampled_")[0]
+    use_sample_id = min_sample_id > 0 or max_sample_id < 99
```

Comment:
what is this part supposed to do?

should it be something like this to check the sample ID range?

```suggestion
    use_sample_id = min_sample_id >= 0 and max_sample_id <= 99
```

Reply:
This wants to find whenever a range smaller than 100% of the samples is being used. Otherwise we don't need to  `use_sample_id`. Does that make sense?

Reply:
I see. Might be worth adding a comment